index.js

```diff
@@ -8,6 +8,7 @@ var defaults = {
   trustAzureHeader: false,
   port: 443,
   hostname: null,
+  skipPort: false,
```

Comment:
I see you made it optional. That is even better than I originally thought.

Anyway I think we can change it to `skipDefaultPort`, make it `true` by default (I guess most of the people will probably want to have this turned on) and most importantly *make it work only with 443* since it can't work with any other port.

What do you think?